src/models/hotel/roomavailability-models/roomAvailabilityResponse.ts

```diff
@@ -13,7 +13,7 @@ export interface Rate {
   currency: string
   ratePlan: string
   rateType: string
-  isCancellable: boolean
+  iscancellable: boolean
```

Comment:
better to keep it `isCancellable` it self.

Reply:
from backend its coming as iscancellable.  This was showing wrong in UI, after changing got corrected.. Or else we may have to change backend and proto

Reply:
ok.fine